pkg/db/v1beta1/postgres/postgres.go

```diff
@@ -0,0 +1,206 @@
+/*
+Copyright 2022 The Kubeflow Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package postgres
+
+import (
+	crand "crypto/rand"
+	"database/sql"
+	"fmt"
+	"math/big"
+	"math/rand"
+	"os"
+	"time"
+
+	_ "github.com/lib/pq"
+
+	v1beta1 "github.com/kubeflow/katib/pkg/apis/manager/v1beta1"
+	"github.com/kubeflow/katib/pkg/db/v1beta1/common"
+	"github.com/kubeflow/katib/pkg/util/v1beta1/env"
+	"k8s.io/klog"
+)
+
+const (
+	dbDriver = "postgres"
+
+	connectInterval = 5 * time.Second
+	connectTimeout  = 60 * time.Second
```

Comment:
Should we move these constants to [`pkg/db/v1beta1/common/const.go`](https://github.com/kubeflow/katib/blob/8d58b0a53e35bdea7c1442dc0500a7e3cfa05ad3/pkg/db/v1beta1/common/const.go) to commonalize?

Reply:
That makes sense for now :). Since we didn't inspect any detailed case for connection in both database. I'll commonize it.